src/EPR.Calculator.API/Controllers/CalculatorController.cs

```diff
@@ -14,13 +14,13 @@ namespace EPR.Calculator.API.Controllers
     [Route("v1")]
     public class CalculatorController : ControllerBase
     {
-        private readonly ApplicationDBContext _context;
+        private readonly ApplicationDBContext context;
```

Comment:
private variable name should start with _? to make consistence with others?

Reply:
We should be using this.context everywhere. underscore is not the recommended usage

Reply:
Okay that's fine